tools/tpm2_quote.c

```diff
@@ -212,6 +214,81 @@ UINT16  calcSizeofTPMT_SIGNATURE( TPMT_SIGNATURE *sig )
     return size > sizeof(*sig) ? sizeof(*sig) : size;
 }
 
+FILE* open_file(const char *path)
+{
+    FILE *fp = fopen(path,"w+");
+
+    if(NULL == fp)
+    {
+        printf("OutFile: %s Can Not Be Created !\n",path);
+    }
+
+    return fp;
+}
+
+bool write_file(FILE *fp, void *data, size_t len,
```

Comment:
files.h has routines for these

Reply:
changed the call inside write_file(). Kept the function, because of the added error output handling present there.